openpype/settings/entities/schemas/projects_schema/schema_project_maya.json

```diff
@@ -101,31 +99,57 @@
                     ]
                 },
                 {
-                    "key": "colorManagementPreference",
-                    "type": "dict",
-                    "label": "Color Management Preference (legacy)",
+                    "type": "collapsible-wrap",
+                    "label": "<b>[Deprecated] please migrate all to 'Workfile' and enable it.</b>",
```

Comment:
Does this mean that with Workfile setting enabled it does not support Maya v1 color management (which I believe was also OCIO but used OCIO v1.0) and thus does not support Maya below version 2022?

Reply:
It will also support v1 settings. The code will solve this.